src/Camille.Core/Camille.Core.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\common.csproj.xml" />
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.1;net45;net461;netstandard2.1;netcoreapp3.1;net6.0;net7.0</TargetFrameworks>
+    <TargetFrameworks>net45;net461;netstandard2.1;netcoreapp3.1;net6.0;net7.0</TargetFrameworks>
```

Comment:
maybe you could also remove the `netcoreapp3.1`(End of support: December 13, 2022) and `net7.0`(End of support: May 14, 2024) and add `net8.0`

Reply:
Aight sounds good (does adding net8.0 do anything if everything else stays the same?)

Reply:
Actually I'll do this in a separate commit

Reply:
Actually actually Ill do it here since `<HintPath>$(NuGetPackageRoot)\microsoft.netframework.referenceassemblies.net45\1.0.3\build\.NETFramework\v4.5\System.Web.dll</HintPath>` code scares me

Reply:
> Aight sounds good (does adding net8.0 do anything if everything else stays the same?)

adding net8.0 won't change anything but I think it's a good idea to include the latest LTS version of .NET